assessmentModel/src/commonMain/kotlin/survey/SurveyRule.kt

```diff
@@ -134,11 +135,11 @@ fun JsonElement.compareTo(value: JsonElement?, operator: SurveyRuleOperator, acc
     }
 }
 
-internal fun JsonLiteral.compareTo(value: JsonLiteral, operator: SurveyRuleOperator, accuracy: Double) : Boolean
```

Comment:
JsonLiteral is no longer publicly available, and 'body' is no longer a val. We only have access to 'content' on JsonPrimitive, which is the string representation of the value. This means we are going to need a different approach for doing comparisons.

Reply:
This is now fixed.

Reply:
Thanks for fixing this. I think that we are going to want to move to using our own Json "any" class with custom serializers b/c that was a brutal migration. That's what I ended up doing for the Swift implementation and it allowed me to have a simple class object that matches Json schema draft 7 documentation w/ equatable, serializable, etc. It will also simplify using Kotlin native for Bridge 2.0 on both platforms.